cogs/twitchlisten.py

```diff
@@ -16,22 +17,11 @@ def __init__(self, bot):
         self.is_live = False
         self.last_seen_live = False
 
-        # channels to send to
-        base_path = (Path(os.path.abspath(__file__)) /'..' /'..').resolve()
-        self.config_path = base_path / 'config_data.json'
-        with open(self.config_path) as json_file: # add path thing
-            config_data = json.load(json_file)
-            self.channels = config_data['twitch_announcement_channels']
-
-
-        
-        # twitch and discord channel info 
+        # twitch channel info
         load_dotenv()
         self.client = os.getenv("TWITCH_CLIENT")
         self.secret = os.getenv("TWITCH_SECRET")
         self.user = os.getenv("TWITCH_USER")
-        self.channel = int(os.getenv("TWITCH_DISCORD_CHANNEL"))

```

Comment:
We don't need the corresponding function in `env.py` and the env var in `README` then.

Reply:
This should probably be refactored / edited to use the functions in env.py

Reply:
This is already done in `main` so it will be picked up when rebasing on latest `main`.

Reply:
Oh I also already mentioned that in https://github.com/Splatoon-Stronghold/cas-stronghold-bot/pull/11#discussion_r1713066875.

Reply:
Please also do this - remove the function from `env.py` and the corresponding variable from the `README`. I think the other comments may have been a bit confusing here, sorry for that.

Reply:
Yeah, I believe the channel is supposed to be in the README

Reply:
It's now obtained from config rather than an environment variable according to https://github.com/Splatoon-Stronghold/cas-stronghold-bot/pull/11/files#diff-60aba5e81852268be06573c3a09d1fe4ab8016d488b0ca31af40d49dd27c60efR15, so I think the mentions of this environment variable need to be removed.